flake.nix

```diff
@@ -123,6 +123,7 @@
   ############################################################################
 
   inputs = {
+    nixpkgs-unstable.url = "nixpkgs/nixos-unstable";
```

Comment:
What's the difference between `.url` and `.follows`?

Reply:
![image](https://user-images.githubusercontent.com/1009751/226328646-d48f289f-1c85-47bd-8eb0-e0a890777536.png)

It means that flake will use whatever nixpkgs-unstable input is declared by haskellNix. My workaround is bringing in `nixpkgs-unstable` directly, this allows to have a newer version of it.